pkg/core/tekton/tekton.go

```diff
@@ -292,8 +326,10 @@ func generatePipelineRun(p *v1beta1.Pipeline, codeset domain.Codeset) (*v1beta1.
 			switch param.Name {
 			case codesetNameParam:
 				prb.Param(param.Name, codeset.Name)
-			case codesetVersion:
+			case codesetVersionParam:
```

Comment:
Is this a pipeline run bug you fixed here ? Wasn't this actually used to clone the git repo ? and if so, how did that work before ?

Reply:
Yes. It was working before because the pipeline has default value for codeset version (`main`).

Reply:
Got it, thanks